vllm/envs.py

```diff
@@ -246,6 +246,10 @@
     # Pad the weight for moe kernel or not
     "VLLM_MOE_PADDING":
     lambda: bool(int(os.getenv("VLLM_MOE_PADDING", "1"))),
+
+    # If set, vllm will print verbose logs during installation
+    "VLLM_USE_TRITON_AWQ":
+    lambda: bool(int(os.getenv("VLLM_USE_TRITON_AWQ", '0'))),
```

Comment:
Can we do this as:

lambda: (os.getenv("VLLM_USE_TRITON_AWQ", "True").lower() in ("true", "1"))

(Make the default "True", to favor Triton.)

Reply:
Oh yes, that makes sense for our repo.